lib/manageiq/api/client.rb

```diff
@@ -8,6 +8,7 @@
 require "query_relation"
 
 require "manageiq/api/client/client"
+require "manageiq/api/client/exception"
```

Comment:
minor nitpick, I assumed the filename should be plural, and then confirmed rails does plural, like: https://github.com/rails/rails/blob/master/actionpack/lib/action_controller/metal/exceptions.rb